app/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html ng-app="cowsayApp">
+  <head>
+    <title>Geoff's Cowville</title>
+  </head>
+  <body>
+    <main>
+      <section class="container" ng-controller="CowsayController">
+        <h2>{{ cowsayCtrl.title }}</h2>
+        <pre class="cow">
+          {{ cowsayCtrl.speak(cowsayCtrl.text }}
```

Comment:
Thanks. It seemed like it was kind of working anyways, but that's a good catch. I'll fix it up and you can check it out.